src/dsp_tools/models/custom_warnings.py

```diff
@@ -19,7 +24,16 @@ class DspToolsUserWarning(Warning):
     @classmethod
     def showwarning(cls, message: str) -> None:
         """Print the warning, without context"""
-        cprint(f"WARNING: {message}", color="red", attrs=["bold"])
+        print(BOLD_RED + f"WARNING: {message}" + RESET_TO_DEFAULT)
```

Comment:
Interestingly it does not work with cprint. You have to configure them with ANSI, but I think this way it is tenable for future use.